vnext/src/Libraries/Components/Button.windows.js

```diff
@@ -312,29 +321,72 @@ class Button extends React.Component<ButtonProps> {
     const Touchable = TouchableHighlight;
     //  Platform.OS === 'android' ? TouchableNativeFeedback : TouchableOpacity;
     // Windows]
-    return (
-      <Touchable
-        accessibilityLabel={accessibilityLabel}
-        accessibilityRole="button"
-        accessibilityState={accessibilityState}
-        hasTVPreferredFocus={hasTVPreferredFocus}
-        nextFocusDown={nextFocusDown}
-        nextFocusForward={nextFocusForward}
-        nextFocusLeft={nextFocusLeft}
-        nextFocusRight={nextFocusRight}
-        nextFocusUp={nextFocusUp}
-        testID={testID}
-        disabled={disabled}
-        onPress={onPress}
-        tabIndex={tabIndex}
-        touchSoundDisabled={touchSoundDisabled}>
-        <View style={buttonStyles}>
+    // [Windows
+    if (Platform.OS === 'windows') {
+      return (
+        <Touchable
+          accessibilityLabel={accessibilityLabel}
+          accessibilityRole="button"
+          accessibilityState={accessibilityState}
+          hasTVPreferredFocus={hasTVPreferredFocus}
+          nextFocusDown={nextFocusDown}
+          nextFocusForward={nextFocusForward}
+          nextFocusLeft={nextFocusLeft}
+          nextFocusRight={nextFocusRight}
+          nextFocusUp={nextFocusUp}
+          testID={testID}
+          disabled={disabled}
+          onPress={onPress}
+          tabIndex={tabIndex}
+          touchSoundDisabled={touchSoundDisabled}
+          underlayColor={PlatformColor('SystemBaseMediumLowColor')}
+          style={
+            this.state.hover
+              ? [
+                  buttonStyles,
+                  {
+                    backgroundColor: PlatformColor('SystemListLowColor'),
```

Comment:
Are there button/control specific colors we could use instead of SystemListLowColor?

Reply:
A concern is that even if the platform colors match up now, if they are for different bits of UI, they can change in the future.

From the name I think this color might be for a list control.

Reply:
Originally I tried using the Button Resources found [here](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Controls.Button?view=winrt-19041) such as ButtonBackground, ButtonBackgroundPointerOver, etc. but the colors produced did not match the color styling displayed in the Xaml Controls Gallery for Button or the design spec for the XAML button. Let me check in with some people to see how XCG is getting different colors.

Reply:
Alright got to the bottom of it. Essentially the coloring of the Button control is different for MUX 2.5 vs WinUI 2.5. Right now playground uses MUX 2.5 as its default for XAML leading to the coloring being different from what is shown in the XAML Controls Gallery which is running on WinUI 2.5. Brush names are the same for both MUX and WinUI just target different colors.